.github/workflows/build_and_test_galactic.yaml

```diff
@@ -0,0 +1,74 @@
+# This config uses industrial_ci (https://github.com/ros-industrial/industrial_ci.git).
```

Comment:
Right now these files include a lot of duplication. Not a big deal but we should probably consolidate these in the future.

Reply:
I can probably clean them up and merge files.

Reply:
But if I merge them in one file (build_and_test.yaml would contain foxy, rolling and galactic). We won't be able to create different badges.